controllers/health/shutdownHandler.go

```diff
@@ -0,0 +1,48 @@
+package health
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/cloudogu/k8s-dogu-operator/api/ecoSystem"
+	v1 "github.com/cloudogu/k8s-dogu-operator/api/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+type ShutdownHandler struct {
+	doguInterface ecoSystem.DoguInterface
+}
+
+func NewShutdownHandler(doguInterface ecoSystem.DoguInterface) *ShutdownHandler {
+	return &ShutdownHandler{doguInterface: doguInterface}
+}
+
+func (s *ShutdownHandler) Start(ctx context.Context) error {
+	logger := log.FromContext(ctx).WithName("shutdown handler")
```

Comment:
```suggestion
	logger := log.FromContext(ctx).WithName("health shutdown handler")
```

Maybe be more specific. There could be other shutdown handlers in the future.